src/AciTypeResolver.js

```diff
@@ -140,6 +142,18 @@ class AciTypeResolver extends TypeResolver {
 
         return [typeDef, vars]
     }
+
+    getContractAndFunctionNameById(functionId) {
```

Comment:
`getFunction` should fit better, have a function name that includes all parameters and return structure properties makes no sense to me, see BytecodeTypeResolver for example